### PR TITLE
Add my profile

### PR DESCRIPTION
**Pull request by ste001**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1560"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

File: public/data/ste001.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://twitter.com/FearTheSock",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/stefano-merazzi",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Gitlab",
```
Comment (Panquesito7) [inline public/data/ste001.json:23]: I recommend capitalizing the `l` here. What do you think?
```suggestion
      "name": "Follow me on GitLab",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Comment (ste001) [reply public/data/ste001.json:23]: Yeah, works better since the actual name is "GitLab". I'll update ASAP

Comment (ste001) [reply public/data/ste001.json]: Done

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**